content/800-data-platform/400-platform-console/maturity-levels.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: 'Maturity levels'
+metaTitle: 'Maturity levels'
+metaDescription: ''
+---
+
+<TopBlock>
+
+Prisma releases updates to the Prisma Data Platform multiple times per week, as opposed to the Prisma ORM that we release on a set schedule every few weeks. This is why we consider the lifecycle and process for maturing features in the Prisma Data Platform differently.
+
+You can [check out the releases and maturity process for the Prisma ORM](/about/prisma/releases) for further details.
+
+</TopBlock>
+
+##
+
+### Early Access
+
+If a feature on the Prisma Data Platform is labeled as **Early Access**:
+
+- We have validated a problem and are considering a solution to it but are not certain whether that solution is complete or a perfect fit.
+- We want to gather more feedback and adjust the solution as necessary, knowing that users are prepared for significant breaking changes
+- We reserve ourselves the right to throttle or remove access to a feature in Early Access to preserve the stability of the platform, or enforcing its use to stay within the scope defined in our [Terms of Service](https://pris.ly/privacy).
+
+As always, your feedback in our [Slack](https://slack.prisma.io/) or [Discord](https://pris.ly/discord) is invaluable to shape the design of the features. This will help us ensure that they can solve your problems in the best way possible.
+
+### Preview
+
+If a feature on the Prisma Data Platform is labeled as **Preview**:
+
+- We have refined the software based on the valuable feedback we obtained during the Early Access phase.
+- We developed the feature further, bringing it closer to the final version, though it's not completely ready for production usage.
+- We have lifted the invite gate, so users no longer need an invitation to access the feature. Users just need to sign up to gain access.
+- We have increased the stability of the software compared to the Early Access phase. While there might still be some issues, they should be less frequent and less critical.
+- We use the Preview phase as a final stress test to ensure the product is ready for heavy production workloads.
+
+We recommend testing the product in a staging environment and welcome any feedback in our [Slack](https://slack.prisma.io/) or [Discord](https://pris.ly/discord). This will assist us in improving the product for its final release.
+
+### General Availability
+
+If a feature in the Prisma Data Platform is Generally Available:
+
+- The solution has been tested for some time and we received enough feedback to consider it stable and ready for production use.
+- The feature meets our [availability and SLO standards](./limits-and-status#availability-and-slo).
```

Comment:
```suggestion
- The solution has undergone extensive testing and, based on significant feedback, is deemed stable and ready for production use cases.
```

Reply:
I want to remove SLOs and SLAs whilst this entire release/launch process is under review.